src/reference.js

```diff
@@ -37,7 +37,7 @@ export default class Reference {
      *        (in which case its .context() is used here).
      */
     constructor(type, referencedContext) {
-        if (API_CONFORMANCE_CHECKS) {
+        if (process.env.NODE_ENV === 'debug') {
```

Comment:
Babel has a plug-in for expanding out environment variables, but not one for arbitrary global constants. Also `NODE_ENV` is a rather standard. It made sense to use this check instead of `API_CONFORMANCE_CHECKS`.